### PR TITLE
Compendium/weapons

### PR DESCRIPTION
**Pull request by WaakFin**
Adds most weapons (all those available for purchase + most rares) from Wasteland Wares list to the compendium.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -110,8 +110,75 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+  - Silaha Plasma "Igneous"
+  - Silaha Plasma "Vulcan"
+  - Sledgehammer
+  - Sparkle-Nade
+  - Spear
   - Staff
+  - Super Sledge
+  - Thermic Lance
+  - Warhammer
+  - Yao Guai Gauntlets
```
Comment (kmoschcau) [inline CHANGELOG.md:181]: This needs to go under the heading of the next release ("Unreleased") version. And only the ones you added new.

File: CHANGELOG.md [hunk 2]
```diff
@@ -140,6 +207,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - updated the rendering of weapon attack messages, providing a more polished
   experience and only showing condensed info, that can be expanded on demand
   ([#58](https://github.com/Wasteland-Ventures-Group/WV-VTT-module/issues/58))
+- Item attribute changes
+  - Fire Axe: Throw -attack and range
+  - Lumber Axe: Throw -attack and range
```
Comment (kmoschcau) [inline CHANGELOG.md:212]: This also needs to go under the "Unreleased" heading.

File: src/compendiums/item/weapon/fire_axe.json [hunk 3]
```diff
@@ -15,13 +15,30 @@
...
           }
+        },
+        "Throw": {
+          "ap": 6,
+          "damage": {
+            "base": 3,
+            "dice": 2,
+            "diceRange": true
+          },
+          "rounds": 1
```
Comment (kmoschcau) [inline src/compendiums/item/weapon/fire_axe.json:26]: Adding a new attack should be communicated in the changelog as well.

**Review by kmoschcau — CHANGES_REQUESTED** [review]
A few changes are needed, mostly relating to the changelog.

File: CHANGELOG.md [hunk 4]
```diff
@@ -11,6 +11,84 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+  - Hunting Bow
+  - IF-44 "Angel Bunny"
+  - IF-84 "Stampede"
+  - IF-92 "Melrose"
+  - IF-G12 "Champion"
+  - Improvised Melee (small)
+  - Longbow
+  - Lumber Axe
+  - Mace
+    - Manticore Gauntlets
```
Comment (kmoschcau) [inline CHANGELOG.md:53]: The indent here is wrong. It should be one less.

File: CHANGELOG.md [hunk 5]
```diff
@@ -11,6 +11,84 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- missing notes on weapons
+  - IF-P39 "Troubleshoes"
+  - Zebra Rifle
+- new attack "Throw" for weapons:
+  - Fire Axe
+  - Lumber Axe
+- attribute Ranges for weapons:
+  - Fire Axe
+  - Lumber Axe
+
```
Comment (kmoschcau) [inline CHANGELOG.md:91]: The extra line should be removed.

File: CHANGELOG.md [hunk 6]
```diff
@@ -112,9 +190,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - weapons
   - Machete
   - Staff
-- missing notes on weapons
-  - IF-P39 "Troubleshoes"
-  - Zebra Rifle
+
```
Comment (kmoschcau) [inline CHANGELOG.md:193]: The extra line should be removed.

**Review by kmoschcau — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (WaakFin) [reply CHANGELOG.md]: Removed extra indent

File: CHANGELOG.md (hunk 5 above)
Comment (WaakFin) [reply CHANGELOG.md:91]: Removed in latest commit

File: CHANGELOG.md (hunk 6 above)
Comment (WaakFin) [reply CHANGELOG.md:192]: Removed in latest commit

**Pull request merged**